build/pipelines/templates/build-console-public.yaml

```diff
@@ -0,0 +1,43 @@
+# build xamlStyler.Console nuget package
+jobs:
+- job: BuildConsoleNuget
+  displayName: 'Build XamlStyler.Console Nuget Package'
+  pool:
+    vmImage: 'ubuntu-latest'
+
+  steps:
+  - task: UseDotNet@2
+    displayName: 'Use .NET Core 3.1 sdk'
+    inputs:
+      packageType: sdk
+      version: 3.1.x
+      installationPath: $(Agent.ToolsDirectory)/dotnet
+
+  # The first task is the dotnet command build, pointing to our csproj file
+  - task: DotNetCoreCLI@2
+    displayName: 'dotnet build'
+    inputs:
+      command: 'build'
+      arguments: '--configuration $(buildConfiguration)'
+      projects: 'XamlStyler.Console/XamlStyler.Console.csproj'
+
+# The second task is dotnet pack command again pointing to the csproj file
+# The nobuild means the project will not be compiled before running pack, because its already built in above step
+  - task: DotNetCoreCLI@2
+    displayName: 'dotnet pack'
+    inputs:
+      command: 'pack'
+      packagesToPack: '**/XamlStyler.Console.csproj'
+      arguments: '--configuration $(buildConfiguration)'
+      versioningScheme: 'off'
+      nobuild: true
+      majorVersion: '$(major)'
+      minorVersion: '$(minor)'
+      patchVersion: '$(patch)'
+
+  - task: PublishPipelineArtifact@1
```

Comment:
I am not finding this artifact published anywhere. I do see the VSIX (with the right version number), which seems to install and work properly.